hack/local-development/dev-setup-register-gardener

```diff
@@ -336,6 +336,7 @@ $ADMISSION_CONTROLLER_PORT_STRING
     - UPDATE
     resources:
     - managedseeds
+    - managedseedsets
```

Comment:
I think I missed this the last time, but why are `ManagedSeed`s and `ManagedSeedSet`s part of this `validate-resource-size.gardener.cloud` webhook config? I think we only validate resources that are createable/updateable by end-users.

Reply:
OK, removed the entire `seedmanagement` group